code/controllers/subsystems/accounts.dm

```diff
@@ -0,0 +1,73 @@
+SUBSYSTEM_DEF(money_accounts)
+	name = "Money Accounts"
+	wait = 5 MINUTES
+	priority = SS_PRIORITY_MONEY_ACCOUNTS
+	var/list/datum/money_account/all_accounts = list()
+
+	// Independent, globally accessible accounts not tied to a network
+	var/list/datum/money_account/all_glob_accounts = list()
+
+	var/list/datum/money_account/escrow/all_escrow_accounts = list()
+
+	var/list/processing_accounts
+
+	var/adjustment = 0
+
+/datum/controller/subsystem/money_accounts/fire(resumed = FALSE)
```

Comment:
I can see the benefit of that, but since this is looping relatively rarely and performing very few actions, I'm not that worried about performance. I'm also leery of having features which require a working database to function, and that probably wouldn't pass muster upstream.